Create ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md (#2337)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-14-6-device-owner-toggle-will-not-switch-back-after-cancelling.md

---
date: ""

> title: Device owner toggle will not switch back after cancelling

Author: The Claude AI suggested to use the following title:
```suggestion
title: Device owner toggle no longer remains in incorrect state after canceling the change
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3806
version: 1020.14.6
---

> Device owner toggle will not switch back after cancelling

Author: The Claude AI suggested to use the following body:
```suggestion
In the device details view, administrators can assign the device owner. However, when attempting to change the device owner and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.
```

> Device owner toggle will not switch back after cancelling

Reviewer: ```suggestion
In Device management, when attempting to change the device owner on the device "Info" tab and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.

```

> In Device management, when attempting to change the device owner on the device "Info" tab and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.

Reviewer 2: ```suggestion
When attempting to change the device owner on the **Info** tab in the device details and then canceling the action, the toggle control previously remained in an incorrect state and appeared as if the change had been applied. With this fix, canceling the device owner change now correctly reverts the toggle control back to the original state, avoiding confusion about the current device owner assignment.
```
